avm/res/db-for-postgre-sql/flexible-server/tests/e2e/encr-hsm/e2eignore

```diff
@@ -0,0 +1 @@
+The test is skipped, as the HSM test would require a persistent instance to be available and configured at all times, inflicting unreasonable costs for the contributor.
```

Comment:
```suggestion
The test is skipped because running the HSM scenario requires a persistent Managed HSM instance to be available and configured at all times, which would incur significant costs for contributors.
```